types/src/state_store/mod.rs

```diff
@@ -67,6 +67,7 @@ pub enum StateViewId {
     TransactionValidation { base_version: Version },
     /// For test, db-bootstrapper, etc. Usually not aimed to pass to VM.
     Miscellaneous,
+    Replay,
```

Comment:
not sure if this is the right thing to add @msmouse, @gelash   - but otherwise I was getting complaints from speculative logs, as those were not initialized.

Reply:
(I'm not sure; how did the replay-verify tool work?)

Reply:
It's because of this: https://github.com/aptos-labs/aptos-core/blob/e0662930743c280a4b7c51fad94256439a8ac1f7/aptos-move/aptos-vm-logging/src/log_schema.rs#L67

I think adding an extra variant is clean, but question is whether you'd like the speculative logging support for replay. I'd venture to say probably yes? (since there will be speculative aborts - and you don't want to output/observe random logs). In that case, would want to match name to Execution above, not to have Misc - it will complain about initialization but for solving that you'd just need smt like https://github.com/aptos-labs/aptos-core/blob/e0662930743c280a4b7c51fad94256439a8ac1f7/aptos-move/aptos-vm/src/block_executor/mod.rs#L414 (since the debugger block execution goes a different path?)